src/Service.php

```diff
@@ -65,7 +71,13 @@ public function __construct(ConfigInterface $config, $identifier, array $types,
         $this->operations = array();
         $this->types = array();
         foreach ($types as $type) {
-            $this->types[$type->getIdentifier()] = $type;
+            if ($type !== null) {
+                $this->types[$type->getIdentifier()] = $type;
+            }
```

Comment:
@kasperg For WSDLs that have nil/void return types, it will issue a fatal error without this `if`.